### PR TITLE
Update documenation for Tableau

### PR DESCRIPTION
**Pull request by johnathan-rodriguez**

### CODE REVIEW

File: docs/blueprint-library/tableau/tableau-check-refresh-status.md [hunk 1]
```diff
@@ -45,7 +53,7 @@ source:
     TABLEAU_USERNAME: null ## REQUIRED
     TABLEAU_PASSWORD: null ## REQUIRED
     TABLEAU_SERVER_URL: null ## REQUIRED
-    TABLEAU_SITE_ID: default ## REQUIRED
+    TABLEAU_SITE_ID: null ## REQUIRED
```
Comment (wrp801) [inline docs/blueprint-library/tableau/tableau-check-refresh-status.md:56]: I didn't notice this in the code - but the SITE_ID should have default setting of `default`

**Review by wrp801 — CHANGES_REQUESTED** [review]
Can you clarify if you made widespread changes to the Site ID? If not provided, it should resort to `default`

File: docs/blueprint-library/tableau/tableau-check-refresh-status.md (hunk 1 above)
Comment (johnathan-rodriguez) [reply docs/blueprint-library/tableau/tableau-check-refresh-status.md:56]: as far as I can tell reviewing the sanity revisions the default value for TABLEAU_SITE_ID was not in production and there is no default value in argparse for this argument either

Comment (wrp801) [reply docs/blueprint-library/tableau/tableau-check-refresh-status.md:56]: My memory was incorrect, I had it reversed. This was the old logic we had in place:
```
    if str(site_id).lower() == 'default': 
        site_id = ''
```
If a user provides the ‘default’ site it should not be set in the connection [](edited-with-Axolo)

Comment (wrp801) [reply docs/blueprint-library/tableau/tableau-check-refresh-status.md:56]: So I think the docs are good [](edited-with-Axolo)

**Review by wrp801 — APPROVED** [review]

**Pull request merged**